exercises/practice/state-of-tic-tac-toe/state-of-tic-tac-toe_spec.lua

```diff
@@ -65,15 +65,6 @@ describe('state-of-tic-tac-toe', function()
       assert.are.same('win', state_of_tic_tac_toe.gamestate(board))
     end)
 
-    it('finished game where x won via middle row victory', function()
-      local board = {
-        'O O', --
-        'XXX', --
-        ' O ' --
-      }
-      assert.are.same('win', state_of_tic_tac_toe.gamestate(board))
-    end)
```

Comment:
I think this is the test that was manually removed. It must have slipped back in the next time the spec generator was run. We should be protected against this now.

Reply:
That was a different test case I accidentally neglected to remove.

The case I removed: 
```
description = "Invalid boards -> Invalid board"
include = false
```

was essentially the same as
```
description = "Invalid boards -> Invalid board: X won and O kept playing"
```

but with a different error string. (I stopped checking the error strings, anticipating the test case might get regenerated.)


In general, does this track have a preference for or against checking error strings?

Reply:
> In general, does this track have a preference for or against checking error strings?

At least for recent tests I've been checking the strings. I don't feel strongly one way or the other.